src/components/WorkPoolStatusIcon.vue

```diff
@@ -39,23 +39,7 @@
 
 <style>
 .work-pool-status-icon { @apply
-  flex
-  items-center
```

Comment:
content-less div that's inside of an inline element so this doesn't center the icon as-is